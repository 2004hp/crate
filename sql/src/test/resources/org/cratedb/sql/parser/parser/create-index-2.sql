CReaTE INdeX index2 On table2 usING FulLteXt(table2.column2) with (prop1='abc', prop2=true, prop3=12, prop4=12.567)