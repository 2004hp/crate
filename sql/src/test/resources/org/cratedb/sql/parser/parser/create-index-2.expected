org.cratedb.sql.parser.parser.CreateIndexNode@231ae15a
name: index2
statementType: CREATE INDEX
unique: false
indexName: index2
tableName: table2
indexMethod: FulLteXt
indexColumns: 
indexProperties: {prop2=value: true
type: BOOLEAN NOT NULL
, prop1=value: abc
type: CHAR(3) NOT NULL
, prop3=value: 12
type: INTEGER NOT NULL
, prop4=value: 12.567
type: DECIMAL(5,3) NOT NULL
}
existenceCheck: NO_CONDITION
	
	org.cratedb.sql.parser.parser.IndexColumnList@4db50639

	[0]:	
	org.cratedb.sql.parser.parser.IndexColumn@27abb00a
	columnName: column2
	tableName: table2
	ascending
